preppal-fe/src/pages/register.tsx

```diff
@@ -0,0 +1,47 @@
+import Container from 'react-bootstrap/Container';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import Image from 'react-bootstrap/Image';
+
+const logo = require('../assets/logo.png')
+
+function Register() {
+  return (
+    <Container className="d-flex align-items-center justify-content-center" style={{ minHeight: '100vh' }}>
+        <div className="w-100" style={{ maxWidth: '400px' }}>
+            <Card className="p-4" style={{ backgroundColor: "#F2E8DC" }}>
+                <div className="text-center mb-4">
+                    <Image src={logo} alt="Logo" rounded />
+
+                </div>
+                <h2 className="text-center mb-4">Register</h2>
```

Comment:
we should change this to "Create your account"

Reply:
This has been updated in signup.tsx